CONTRIBUTING.md

```diff
@@ -601,7 +594,7 @@ name(node) # get node name
 name!(node, "somename") # set node name
 ```
 
-* When using conditional branching, if code is statement-like, an
+- When using conditional branching, if code is statement-like, an
```

Comment:
A question about this, do you mean that one should not use the common Julia idiom:
```julia
   expression && statement
```
or
```julia
    expression &&
        statement
```
and the same but with `||`.
This is very common for simple checks, such as `isvalid(foo) || throw(ArgumentError("$foo is not valid"))`